### PR TITLE
Review: review of past project

### PR DESCRIPTION
**Pull request by Vgonma**
# Files added
 - index.html
 - style.css
# Reviews required
 - Linters
 - Semantics
 - Best practice

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="styles.css" rel="stylesheet">
+    </head>
+    <body>
+        <nav id="navbar" class="nav">
+            <a href="#projects"></a>
+        </nav>
+    <div id="welcome-section">
+        <h1>Welcome!</h1>
+    </div>
```
Comment (Vgonma) [inline index.html:15]: Looking good 👍  but remember to use semantic tags such as `section` in place of your `div` s. Semantic tags make your code easier to read.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#navbar{
+    position:fixed;
+    top:0;
+}
```
Comment (Vgonma) [inline style.css:4]: This works but 💡 remember 💡  to check linter errors such as one space between '#navbar' and ' { ', also only use two spaces indentation for 'position' and 'top'. Linters always help you make your code cleaner and easier to understand and review by others. 😄

File: style.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+@media only screen and (min-width: 500px) {
+body {
+  background-color:tomato;
+}
+}
```
Comment (Vgonma) [inline style.css:9]: Nice job coding this! 👍. Remember to always add indentation where it is needed. Your body selector and all the elements it holds should have an indentation to show it is inside of the '@media' query. Having good indentation will make it easier for anyone to read and make changes to your code. 👀

**Review by Vgonma — COMMENTED** [review]
# Changes required 🔍 
## Well done so far, only a little bit more. 
Please review the comments left on each section down below and make the required changes. 
If anything is unclear tag me in a comment and I will explain.
Good luck and happy coding! 😄

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @Vgonma,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Very polite and professional comments. ✔️
 - Explanatory comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Remember that if the linter checks are not passing in the Github PR the project should be invalidated. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**